bulk_index.py

```diff
@@ -46,7 +46,7 @@ def populate(print_indexable: bool, paper_id: str, id_list: str,
     index_chunk_size = 250
     chunk: List[str] = []
     meta: List[DocMeta] = []
-
+    index.current_session().create_index()
```

Comment:
Presumably this just fails if the index already exists?

Reply:
* fails gracefully

Reply:
Yes